emission/net/api/stats.py

```diff
@@ -46,22 +46,33 @@ def storeClientEntry(user, key, ts, reading, metadata):
   currEntry = createEntry(user, key, ts, reading)
   # Add the os and app versions from the metadata dict
   currEntry.update(metadata)
-  get_client_stats_db().insert(currEntry)
+  response = get_client_stats_db().insert(currEntry)
```

Comment:
Also, if there is an exception, maybe continue logging to the old method (in mongodb) so that we can repair when we are done.